docs/test-plan.md

```diff
@@ -0,0 +1,248 @@
+# Test Plan: lockbox-extension
+
+## Ownership
+
+* Product Integrity: Matt Brandt
+* Product Manager: Sandy Sage
+* Engineering Manager: Devin Reams
+* Engineering Leads: Matt Miller & Jim Porter
+
+## Purpose
+
+Test scope for Q3 and Q4 of the Lockbox Firefox extension. Q3 will take the project through the internal Alpha phase of the product to Q4 when the team will tune a product that we can ship as a limited Beta to external users via Test Pilot.
+
+### Entry Criteria
+
+* QA has access to all PRDs, mocks, and related documents.
+* A run script that installs the WebExtension for Firefox Nightly
+* A compiled and signed extension for Firefox Beta and Release
+* Staging and release channels
+
+### Exit Criteria
+
+* All bugs against the features have been triaged
+* All P1 and P2 bugs have been resolved fixed and verified by Product Integrity
+* Sign off criteria have been met
+
+### Acceptance Criteria
+
+This section broadly defines when the product is ready to ship
+
+* QA has signed off
+* Product confirms we are feature complete
+* Engineering confirms we are feature complete
+* a11y-review sign off
+* Sec review completed and sign off
+* All info is localized at least for a pre-defined set of locales 
+* All telemetry requirements are in place
+* All the necessary PR and blog posts have been sent out 
+
+## Overall Status
+
+Alpha - estimated Q3 timeline
+
+* July: lay the foundation for system add-on with embedded WebExtension development
+* August: begin secure storage and front-end editor design and implementation
+* September: finish editor and storage, implement lock/unlock, package for release
```

Comment:
Two of these were implemented in a way users would experience them.  I would say that lock/unlock is partially complete from a user's perception.

Reply:
r+ Changed section header to `Timelines and Milestones`. Also removed lock/unlock.